tests/test_snmp_notification_receiver.rb

```diff
@@ -100,7 +100,7 @@ def test_create_destroy_single_ipv6
     assert_equal('informs', Cisco::SnmpNotificationReceiver.receivers[id].type)
     assert_equal('ab', Cisco::SnmpNotificationReceiver.receivers[id].username)
     assert_equal('3', Cisco::SnmpNotificationReceiver.receivers[id].version)
-    assert_equal('red', Cisco::SnmpNotificationReceiver.receivers[id].vrf)
+    assert_equal('red', Cisco::SnmpNotificationReceiver.receivers[id].vrf) # rubocop:disable Metrics/LineLength
```

Comment:
This line is shorter than lines around it, was rubocop complaining about its length?